### PR TITLE
[ZH] Add `Gameplay/Health`

### PR DESCRIPTION
**Pull request by Andy-K-Sparklight**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
---
Translated `Gameplay/Health/en.md` to `Gameplay/Health/zh.md`

Some parts of the original page are not easy to translate. I've personally chosen expressions that fit the original text most. Please point out if there are better translations!
原文有些句子不太容易翻译，个人选择了最贴合原文的表达方法，如果有更好的翻译请不吝指出！

### CODE REVIEW

File: wiki/Gameplay/Health/zh.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+tags:
+  - 生命值
+  - 生命
+  - 血量
+---
+
+# 血量
+
+**血量**（HP）是用来判定玩家有没有通过一张[谱面](/wiki/Beatmap)的数据。血量以[血条](/wiki/Client/Interface/Health_bar)的形式显示，并且会在游戏时不断变化。[掉血速度](/wiki/Beatmapping/HP_drain_rate)决定了保持较高血量的难易程度。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Health/zh.md:10]:
```suggestion
**血量**（HP）是用来判定玩家有没有通过一张[谱面](/wiki/Beatmap)的数值。血量以[血条](/wiki/Client/Interface/Health_bar)的形式显示，并且会在游戏时不断变化。[掉血速度](/wiki/Beatmapping/HP_drain_rate)决定了保持较高血量的难易程度。
```

用数值更为恰当

File: wiki/Gameplay/Health/zh.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+在 [osu!mania](/wiki/Game_mode/osu!mania) 模式下，只有音符（Hit Objects）会影响血量，没有[掉血](/wiki/Beatmapping/Health_drain)机制。
+
+每成功击打一个音符时，相应的评分会显示出来，根据这个评分，总分和血量也会相应变化。准确击打（得分 300）会比糟糕击打（得分 50）回复更多的血量。反过来，漏掉音符会扣血。当按住长条（Hold Notes）时，血量会缓慢回复。
+
+回血和掉血的所有途径如下：
+
+血量回复：
+
+- 打出 **彩 300（Rainbow 300）**，**300** 或 **200**（回复的血量从多到少）
```
Comment (YumeMuzi) [inline wiki/Gameplay/Health/zh.md:96]:
```suggestion
- 打出 **彩 300**，**300** 或 **200**（回复的血量从多到少）
```

彩 300 在 cn osu!mania 的传播较广，可以不用标出原文

Comment (YumeMuzi) [inline wiki/Gameplay/Health/zh.md:96]:
```suggestion
- 打出 **彩 300**，**300** 或 **200**（回复的血量从多到少）
```

彩 300 在 cn osu!mania 的传播较广，可以不用标出原文

File: wiki/Gameplay/Health/zh.md (hunk 1 above)
Comment (YumeMuzi) [inline wiki/Gameplay/Health/zh.md:10]:
```suggestion
**血量**（HP）是用来判定玩家有没有通过一张[谱面](/wiki/Beatmap)的数值。血量以[血条](/wiki/Client/Interface/Health_bar)的形式显示，并且会在游戏时不断变化。[掉血速度](/wiki/Beatmapping/HP_drain_rate)决定了保持较高血量的难易程度。
```

用数值更为恰当

**Review by YumeMuzi — APPROVED** [review]
excellent.
非常精彩的翻译。

**Comment by Andy-K-Sparklight** [comment]
Thanks for reviewing. I've commited the changes and updated branch.
多谢！我已经合并这些建议，同时更新了分支。@YumeMuzi

**Review by YumeMuzi — APPROVED** [review]

**Review by TicClick — APPROVED** [review]
thank you. one note, please don't remove original tags (appending their translations, or anything that will help speakers find the article, is the way)

**Pull request merged**